.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -12,6 +12,7 @@ This issue is tracking the tasks that should be implemented **after** the Kubern
 ## Tasks
 
 Prerequisites:
+
 * [ ] Decide which Cluster API release series will support the new Kubernetes version
```

Comment:
Let's drop this prereq. We have a clearly documented policy that states we try to cherry-pick back to 1 minor version.

The only reason we wouldn't do it, is if we need huge changes to CAPI to support it. But that's not realistic and we probably never know it before we actually start the work.